stores/metadata.go

```diff
@@ -2058,6 +2058,10 @@ LIMIT ?
 				return err
 			} else if err = tx.Exec("CREATE TEMPORARY TABLE src AS ?", healthQuery).Error; err != nil {
 				return err
+			} else if err = tx.Exec("CREATE INDEX src_id ON src (id)").Error; err != nil {
+				return err
+			} else if err = tx.Exec("CREATE INDEX src_health ON src (health)").Error; err != nil {
```

Comment:
I'm a little bit sceptical this one is effectively used. Did you confirm this by investigating the query plan?

Reply:
Yeah you are right. I don't think it's needed.